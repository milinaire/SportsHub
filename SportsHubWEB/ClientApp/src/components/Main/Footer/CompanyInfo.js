import React, { Component, Fragment } from "react";
import "./Footer.css";
import {BaseLayout} from "../Layout/BaseLayout";

export class CompanyInfo extends Component {
  state = {
    Headline: "some headline from db",
    Text: "text from db",
  };
  render() {
    return (
      <Fragment>
        <BaseLayout>
          <div>
            {this.props.match.params.name}
            <h1>{this.state.Headline}</h1>
            <p>{this.state.Text}</p>
          </div>
        </BaseLayout>
      </Fragment>

    );
  }
}
