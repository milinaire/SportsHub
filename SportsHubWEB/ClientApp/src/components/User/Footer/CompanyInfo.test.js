
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import {CompanyInfo} from "./CompanyInfo";



describe('Main\Footer\CompanyInfo test', () => {
  it('Main\Footer\CompanyInfo should match snapshot', () => {
    const component = renderer.create(<CompanyInfo
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
