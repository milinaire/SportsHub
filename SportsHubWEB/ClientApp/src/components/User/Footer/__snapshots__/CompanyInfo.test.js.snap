// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainFooterCompanyInfo test MainFooterCompanyInfo should match snapshot 1`] = `
<div
  className="main-part-footer"
>
  <h1>
    Ssome headline from db
  </h1>
  <p>
    text from db
  </p>
</div>
`;
